docs/Reference/Orion-CLI-Syntax.md

```diff
@@ -41,8 +41,7 @@ Displays help and exits.
 
 Displays version information and exits.
 
- 
-## Configuration File 
+### configuration file 
```

Comment:
Is this lower-case c deliberately?  And the change in heading level?

Reply:
No. Fixed now.